app/views/categories/index.html.erb

```diff
@@ -0,0 +1,41 @@
+<div class="header">
+  <%= button_to destroy_user_session_path, class: 'no-styles-button',  method: :delete do %>
+    <i class="fa-solid fa-bars"></i>  
+  <% end %>
+  <h2 class="header-title">Categories</h1>
+  <i class="fa-solid fa-magnifying-glass"></i>
+</div>
+<div class="gray-background">
+  <div class="categories-container">
+    <% @categories.each do |category| %>
+      <%= link_to category_operations_path(category), class: 'no-text-decoration' do %>
+        <div class="category-container">
+          <div class="category-container-left">
+            <% if category.icon %>
+              <%= image_tag "#{category.icon}.png" %>
+            <% end %>
+          </div>
+          <div class="category-container-right">
+            <div class="category-container-data">
+              <div class="d-flex flex-direction-column justify-content-center">
+                <p class="category-container-text"><%= category.name %></p>
+              </div>
+              <div>
+                <p class="no-text-decoration">
+                Total: 
+                </p>
+                <p class="category-container-text"><%= number_to_currency(calculate_total(category.operations))  %><p>
+              </div>
+
+            </div>
+            <hr class="line">
+            <p class="category-container-text"><%= category.created_at.to_s[0...10] %></p>
```

Comment:
- [ ] Please use the correct Typography (font-family, font-weight) while displaying the category amounts on the cards displayed on the category index page just as shown in the Behance design attached below 👇 
### Behance Design
![image](https://github.com/DavidEspinoG/microverse_rails_capstone/assets/46839548/878493d1-645e-4e20-a0f7-1c7e8ac2e54d)
### Your Design
![image](https://github.com/DavidEspinoG/microverse_rails_capstone/assets/46839548/a88993d8-852c-4f8c-ac06-4b09ceee3594)

### Your Design